src/components/front-page-alert/FrontPageAlert.tsx

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { Alert } from '@amsterdam/asc-ui'
+
+import configuration from 'shared/services/configuration/configuration'
+
+import Markdown from '../Markdown'
+
+export const FrontPageAlert = () => {
+  const alertText = configuration.frontPageAlert.text
+
+  return (
+    <>
+      {alertText && (
```

Comment:
Better to make use of an early return based on the existence of the config and location.pathname. (https://medium.com/codex/early-return-in-typescript-explained-along-with-how-i-use-it-in-react-apps-92c0694a8356)

Reply:
done